shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/java/org/apache/shardingsphere/sql/parser/integrate/asserts/statement/dal/DALStatementAssert.java

```diff
@@ -83,6 +86,8 @@ public static void assertIs(final SQLCaseAssertContext assertContext, final DALS
             ShowTableStatusStatementAssert.assertIs(assertContext, (ShowTableStatusStatement) actual, (ShowTableStatusStatementTestCase) expected);
         } else if (actual instanceof ShowIndexStatement) {
             ShowIndexStatementAssert.assertIs(assertContext, (ShowIndexStatement) actual, (ShowIndexStatementTestCase) expected);
+        } else if (actual instanceof SetStatement) {
```

Comment:
`SetStatement` is a new type with `ShowXXXStatement`, it should not between `ShowIndexStatement` and `ShowStatement`, please move `SetStatement` behind `ShowStatement`